easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb

```diff
@@ -0,0 +1,39 @@
+name = 'APBS'
+version = '1.4.1'
+
+easyblock = 'CMakeMake'
+
+homepage = 'http://www.poissonboltzmann.org/apbs'
+description = """ APBS is a software package for modeling biomolecular solvation 
+ through solution of the Poisson-Boltzmann equation (PBE), one of the most popular 
+ continuum models for describing electrostatic interactions between molecular solutes 
+ in salty, aqueous media. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+#git clone -b 1.4.1-binary-release  https://github.com/Electrostatics/apbs-pdb2pqr
+#create your own tarball
```

Comment:
use the official location for sources, see http://sourceforge.net/projects/apbs/files/apbs/apbs-1.4.1/ (via http://www.poissonboltzmann.org/docs/downloads/#downloads) ?